cc_proxy.go

```diff
@@ -98,15 +98,6 @@ func (p *ccProxy) register(pod Pod) ([]IOStream, error) {
 		return []IOStream{}, err
 	}
 
-	for i := 0; i < len(pod.containers); i++ {
-		ioStream, err := p.allocateIOStream()
-		if err != nil {
-			return []IOStream{}, err
-		}
-
-		ioStreams = append(ioStreams, ioStream)
-	}
-
```

Comment:
Not sure I understand: Why are we no longer returning the list of IO streams (eventually the list of tokens) here ?